data/data.go

```diff
@@ -0,0 +1,39 @@
+package data
+
+import (
+	"github.com/go-yaml/yaml"
+	packr "github.com/gobuffalo/packr/v2"
+)
+
+// YML maps the YAML found in _data/data.yml
+// Be sure to update this type when you modify the structure of that file!
+type YML struct {
```

Comment:
Nit: The YML type is only related to yaml in that it uses yaml as a storage medium. Something like `StaticData` or `CircleCIInstallationData` would be more self-describing (though I don't love the names I came up with either).

Reply:
I can't think of a better name either, but went with `YML` because `DataYML` was redundant and failed the linter. Happy to change it later, it's doubtful anyone will depend on this tiny package outside of us (in spite it being public and exported type).